neo/Ledger/Blockchain.cs

```diff
@@ -610,6 +610,21 @@ private void Persist(Block block)
                 foreach (IPersistencePlugin plugin in Plugin.PersistencePlugins)
                     plugin.OnPersist(snapshot, all_application_executed);
                 snapshot.Commit();
+                foreach (IPersistencePlugin plugin in Plugin.PersistencePlugins)
+                {
+                    try
+                    {
+                        plugin.OnCommit(snapshot);
+                    }
+                    catch (Exception ex)
+                    {
+                        if (plugin.ShouldThrowExceptionFromCommit(ex))
+                        {
+                            lastException = ex;
+                        }
+                    }
+                }
+                if (lastException != null) throw lastException;
```

Comment:
We can use `AggregateException` to re-throw all the excetions in `OnCommit()`.

Reply:
Nice suggestion. I’ll implement it.